### PR TITLE
Use rollup instead of webpack. Use jest instead of mocha

### PR DESCRIPTION
**Pull request by eirikbacker**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,51 +4,28 @@
...
-    ],
-    "ignore": [
-      "dist"
-    ]
+    "jest": "23.5.0",
+    "rollup": "0.65.0",
+    "rollup-plugin-buble": "0.19.2",
+    "rollup-plugin-serve": "0.4.2",
+    "rollup-plugin-uglify": "4.0.0",
+    "standard": "11.0.1"
```
Comment (wwalmnes) [inline package.json]: `.npmrc` fil med `save-exact=true` 😄 ?

**Comment by eirikbacker** [comment]
Avoiding WeekMap as we want this to be super backwards compatible :)

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+var bridge = window.nativebridge
```
[the remark this answers is not in the document]
Comment (cristobal) [reply lib/index.js]: Do you still need old school js support then, as far as i can tell the test-suit supports ES5 ¯\_(ツ)_/¯

**Comment by cristobal** [comment]
Good but sometimes we just do ourselves _"bjørnetjeneste"_ 🙂

**Pull request merged**